packages/init/src/steps/install-required-dependencies.ts

```diff
@@ -19,19 +40,8 @@ export async function installRequiredDependencies(
     `${installDevDependencyCommand} ${Utils.CONSTS.SM_PACKAGE_NAME}`
   );
 
-  switch (framework) {
-    case Utils.Framework.FrameworkEnum.react:
-    case Utils.Framework.FrameworkEnum.next:
-      await execCommand(
-        `${installDependencyCommand} ${Utils.CONSTS.PRISMIC_REACT_PACKAGE_NAME}`
-      );
-      break;
-    case Utils.Framework.FrameworkEnum.svelte:
-      await execCommand(
-        `${installDependencyCommand} ${Utils.CONSTS.PRISMIC_DOM_PACKAGE_NAME}`
-      );
-      break;
-  }
+  const deps = depsForFramework(framework);
+  if (deps) await execCommand(`${installDependencyCommand} ${deps}`);
```

Comment:
I know this wasn't in the spec, but do you think we could/should put a spinner or a message to let the user know an action is on going.

Reply:
There's already a spinner, 
if we use execSync the output from the command can be piped to stdout